examples/simplePeriodic.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+    "context"
+    //"http"
+    "net/http"
+    "log"
+    "time"
+    "github.com/mjpitz/go-gracefully/check"
+    "github.com/mjpitz/go-gracefully/health"
+    "github.com/mjpitz/go-gracefully/state"
+)
+
+func main() {
+    monitor := health.NewMonitor([]check.Check{
+        &check.Periodic{
+            Metadata: check.Metadata{
+                Name: "health_check",
+                //Runbook: "https://yourURL/Runbook.md",
+                Weight: 10,
+            },
+            Interval: time.Second * 5,
+            Timeout: time.Second,
+            RunFunc: func(ctx context.Context) (state.State, error) {
+                // make API call, check system health
+                resp, err := http.Get("https://yourURL/health_check")
```

Comment:
It would be good if this was a workable example. Maybe look into using http bin or something similar.